tools/sgcollect_info

```diff
@@ -263,29 +265,46 @@ def make_collect_logs_tasks(zip_dir, sg_url, salt, should_redact):
 
     # Find log file path from logging.["default"] style config
     guessed_logging_path = extract_element_from_default_logging_config('LogFilePath', config_str)
-    if guessed_logging_path:
+    if guessed_logging_path and os.path.isfile(guessed_logging_path):
         # Get the parent directory and the log file name
         log_file_parent_dir = os.path.abspath(os.path.join(guessed_logging_path, os.pardir))
         log_file_name = os.path.basename(guessed_logging_path)
         name, ext = os.path.splitext(log_file_name)
-
-        # iterate over all log files, including those with a timestamp prefix
-        log_file_pattern = "{0}*{1}".format(name, ext)
-        rotated_logs_pattern = os.path.join(
-           log_file_parent_dir,
-           log_file_pattern
-        )
-
-        for log_file_item_name in glob.iglob(rotated_logs_pattern):
-            log_file_item_path = os.path.join(log_file_parent_dir, log_file_item_name)
-            # As long as a task that monitors this log file path has not already been added, add a new task
-            if log_file_item_path not in sg_log_file_paths:
-                print('Capturing rotated log file {0}'.format(log_file_item_path))
-                task = add_file_task(sourcefile_path=log_file_item_path)
-                sg_tasks.append(task)
-
-            # Track which log file paths have been added so far
-            sg_log_file_paths[log_file_item_path] = log_file_item_path
+        # Iterate over all log files, including rotated log files with timestamp in file name.
+        rotated_logs_pattern = (
+            os.path.join(log_file_parent_dir, "{0}*{1}".format(name, ext)),
+            os.path.join(log_file_parent_dir, "{0}*{1}.gz".format(name, ext)))
```

Comment:
`logging."default"` log files (from pre 2.1 SG) don't have compression on rotated logs. That should make this whole bunch of code simpler because we don't have 2 patterns to match on.

Reply:
Yeah, it makes sense, actually it included to keep the same behavior in the old code. But I agree with your suggestion here and it is removed in latest revision.